src/Model/MessageManagerInterface.php

```diff
@@ -14,9 +14,9 @@
 namespace Sonata\NotificationBundle\Model;
 
 use Sonata\CoreBundle\Model\ManagerInterface;
-use Sonata\CoreBundle\Model\PageableManagerInterface;
+use Sonata\DatagridBundle\Pager\PageableInterface;
```

Comment:
Do we already require datagrid bundle in composer.json?

Reply:
Yes, https://github.com/sonata-project/SonataNotificationBundle/blob/3.x/composer.json#L26